### PR TITLE
Add formula for wires webdriver

### PR DESCRIPTION
**Pull request by anthonygreen**
Marionette is an automation driver for Mozilla's Gecko engine. Wires is the new webdriver.
https://developer.mozilla.org/en-US/docs/Mozilla/QA/Marionette/WebDriver

### CODE REVIEW

File: Library/Formula/wires.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  def install
+    puts "#{bin}"
+    system "cargo build"
+    bin.install "target/debug/wires"
+  end
+
+  test do
+    system "#{bin}/wires", "--help"
+  end
+
```
Comment (bfontaine) [inline Library/Formula/wires.rb]: Please remove this empty line.

File: Library/Formula/wires.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class Wires < Formula
+  desc "WebDriver <-> Marionette proxy"
+  homepage "https://github.com/jgraham/wires"
+  url "https://github.com/jgraham/wires/archive/v0.4.2.tar.gz"
+  sha256 "f86fdac8f69f94bd57dc9fdb45851585a24790b069605d033288e419653c9efe"
+
+  depends_on "rust" => :build
+
+  def install
+    puts "#{bin}"
```
Comment (bfontaine) [inline Library/Formula/wires.rb]: You can remove this line.

File: Library/Formula/wires.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    system "cargo build"
```
Comment (bfontaine) [inline Library/Formula/wires.rb]: Please use `system "cargo", "build"`.

**Comment by anthonygreen** [comment]
Fixed the issues with new lines. Outstanding issue is 'GitHub repository not notable enough' although it is linked to from Mozilla's project page https://developer.mozilla.org/en-US/docs/Mozilla/QA/Marionette/WebDriver

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `wires 0.4.2 (new formula)`.

**Pull request closed without merging**